gemsurance.gemspec

```diff
@@ -19,12 +19,12 @@ Gem::Specification.new do |s|
   s.files                     = `git ls-files -- bin lib`.split("\n")
   s.executables               = ["gemsurance"]
 
-  s.add_dependency("bundler", "~> 1.2")
+  s.add_dependency("bundler", ">= 1.2")
```

Comment:
Could you revert this line? The master branch was updated to handle this requirement (with a `< 3.0`).